util/eku.go

```diff
@@ -0,0 +1,17 @@
+package util
```

Comment:
May as well just make this part of `ca.go`. https://github.com/zmap/zlint/blob/master/util/ca.go